mhkit/loads/damage_equivalent_load.m

```diff
@@ -1,34 +1,44 @@
-function DEL=damage_equivalent_load(var,m,options)
+function DEL=damage_equivalent_load(data_signal,m,options)
 
 %%%%%%%%%%%%%%%%%%%%
-%     Calculates the damage equivalent load 
+%     Calculates the damage equivalent load of a single data signal (or channel) 
+%     based on IEC TS 62600-3:2020 ED1. 4-point rainflow counting algorithm from 
+%     fatpack module is based on the following resources:
+%         
+%     - `C. Amzallag et. al. Standardization of the rainflow counting method for
+%       fatigue analysis. International Journal of Fatigue, 16 (1994) 287-293`
+%       `ISO 12110-2, Metallic materials - Fatigue testing - Variable amplitude
+%       fatigue testing.`
+%     - `G. Marsh et. al. Review and application of Rainflow residue processing
+%       techniques for accurate fatigue damage estimation. International Journal
+%       of Fatigue, 82 (2016) 757-765`
 %     
 % Parameters
 % ------------
-%     var: vector  
-%         data of variable/channel being analyzed 
+%     data_signal: vector  
+%         Data signal being analyzed 
 %
 %     m : double or int 
-%         fatigue slope factor of material  
+%         Fatigue slope factor of material  
 %
 %     bin_num : int (optional) 
-%         number of bins for rainflow counting method (minimum=100)
-%         to call: get_DELs(data,chan_info,"binNum",binNum)
+%         Number of bins for rainflow counting method (minimum=100)
+%         to call: get_DELs(data,chan_info,"bin_num",binNum)
 %
 %     data_length : double or int (optional)
-%         Used to control DEL frequency. Length of data in sec. Default for 1Hz is 600 seconds for 10min data
+%         Length of data in sec. Default for 1Hz is 600 seconds for 10min data
 %         to call: get_DELs(data,chan_info,"data_length",t)
 %     
 % Returns
 % ---------
 %     DEL: Structure 
-%         Damage equivalent load of each specified variable
+%         Damage equivalent load of signal
 %
 %         
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 arguments
-    var 
+    data_signal 
     m
     options.bin_num {mustBeNumeric} = 100;
```

Comment:
Is it standard practice for MHKiT-MATLAB to list all function input parameters, even if there isn't [Function Argument Validation](https://www.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html) for the parameter? For example, it looks like `data_signal` and `m` are listed here but not checked.

Reply:
With the arguments method it is typical that if you set the required datatype you also set a default value, making the input optional. I will put it on my to-do list as a bug fix to add more explicit assert statements for variables where needed. Some variables are covered by the asserts in the python functions (i.e. variables that have to be integers). I do generally have checks for structures as well.

Reply:
Also, to address your question, in the assert block is it required that all input arguments be present, even if you are not asserting a datatype.

Reply:
@rpauly18 Thanks for this explanation. Let me know if there's anything I can do to help. Also, we can set up Jenkins to run test on MHKiT-MATLAB if you think it would be beneficial.